helm/chart-operator-chart/values.yaml

```diff
@@ -0,0 +1,21 @@
+# Default values for chart-operator-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+name: chart-operator
+namespace: kube-system
+port: 8000
+
+replicas: 1
+
+image:
+  repository: quay.io/giantswarm/chart-operator
+  tag: [[ .SHA ]]
+
+resources:
+  requests:
+    cpu: 100m
+    memory: 150Mi
+  limits:
+    cpu: 250m
+    memory: 250Mi
```

Comment:
Maybe somebody can provide any insights, if these values seem reasonable.

Reply:
What I know is that we should align requests and limits to have them equal because this makes k8s apparently guarantee that we get what we ask for and some other smart magic I am not fully aware of.

Reply:
+1 what Tim said, I assume this operator runs in the hostcluster? If it does, then the `.Values` stuff will not work like this together with the current Draughtsman.

Reply:
For the first release `chart-operator` will run in the guest cluster and not the host cluster. The operator needs to be self-updating and will be installed in the guest cluster by `cluster-operator`.

See the tech spec for more details.

https://github.com/giantswarm/giantswarm/blob/master/roadmap/MANAGED-SERVICES.md#technical-spec

We still need to sort out the details of how we handle configuration. Aligning with the ` draughtsman .Values` stuff makes sense if possible.

Reply:
I am leaving this for now then?

Reply:
I would say no because we want to have certain guarantees for our system components independent of our own design decisions.

Reply:
I am not sure what to do then. Are you suggesting to refrain from helm templating in guest cluster charts at all, because `draughtsman` also provides `values`?  Also as @rossf7 said we are not using `draughtsman` here.

Reply:
I say we should align the values. The CPU requested should equal the limit. Same for memory.

Reply:
See https://medium.com/google-cloud/quality-of-service-class-qos-in-kubernetes-bb76a89eb2c6.

Reply:
Alright, got it. :+1:

Reply:
Agree. These should be aligned.

Reply:
Done. Just as a side note: This is not the case for some other operators. E.g. aws-operator and flannel

Reply:
I know. We should do this. Feel free to boyscout.